File: src/test/java/coreTDD/passwordVerifier/PasswordTest.java
package coreTDD.passwordVerifier;

import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class PasswordTest {

    @Test
    public void garbageTest() {
        assertThat(true).as("This is a garbage test!").isFalse();
    }
}

File: src/test/kotlin/kotlinCoreTDDTests/passwordVerifier/PasswordTest.kt
package kotlinCoreTDDTests.passwordVerifier

import kotlinCoreTDD.passwordVerifier.Password
import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class PasswordTest {

    @Test
    fun passwordsShorterThan12CharactersAreNotValid() {
        val verificationResult = Password().verify("tooShort")

        assertThat(verificationResult).isFalse()
    }
}